code/modules/mob/mob.dm

```diff
@@ -1487,6 +1487,8 @@ Use this proc preferably at the end of an equipment loadout
 				for(var/atom/A in listed_turf)
 					if(!A.mouse_opacity && !A.name)
 						continue
+					if (istype(A, /atom/movable/light))
+						continue
```

Comment:
Do you really need a special condition for these? Couldn't you just not give them names?

Reply:
well, it's useful to distinguish between the light obj and the "shadows over walls" one, but yeah.